modules/ossm-directing-egress-traffic-through-a-gateway-kubernetes-gateway-api-ambient-mode.adoc

```diff
@@ -0,0 +1,142 @@
+// This procedure is used in the following assembly:
+// * service-mesh-docs-main/gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-directing-egress-traffic-through-gateway-kubernetes-gateway-api-ambient-mode_{context}"]
+= Directing egress traffic through a gateway using the {k8s} Gateway API in ambient mode
+
+[role="_abstract"]
+Use the {k8s} Gateway API and waypoint proxy to direct outbound HTTP traffic through an egress gateway.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator version 3.2 or later.
+
+* You configured the `Istio` and `IstioCNI` resources with ambient profile.
+
+* You have created a `Ztunnel` resource.
+
+.Procedure
+
+. Optional: Enable the {k8} Gateway API custom resource definitions (CRDs). 
```

Comment:
Check if the value of this attribute {k8} in the sentence "Optional: Enable the {k8} Gateway API custom resource definitions (CRDs)."  is supposed to be published as is in the preview?